topics/microbiome/metadata.yaml

```diff
@@ -44,3 +44,6 @@ subtopics:
   - id: metatranscriptomics
     title: "Metatranscriptomics"
     description: "Taxonomic and functional characterisation of mixed samples using transcriptome data."
+  - id: clinical-metaproteomics
```

Comment:
Can we remove the `clinical`?

Reply:
No, we cannot, unfortunately. The subtopic ID must match exactly in both topics, since the tutorials are symlinked in.

Reply:
users will only see the title field, "Metaproteomics", so maybe ok to leave it?

Reply:
@bebatut lets remove the `clinical` in a follow-up if its too confusing.